core/build.gradle.kts

```diff
@@ -3,13 +3,13 @@ apply<PublishPlugin>()
 dependencies {
     api(Modules.query)
 
-    compileOnly(Dependencies.javaPersistenceApi)
-    compileOnly(Dependencies.slf4j)
-    compileOnly(Dependencies.hibernate)
+    compileOnly(libs.java.persistence.api)
+    compileOnly(libs.slf4j)
+    compileOnly(libs.hibernate)
 
     testImplementation(Modules.testFixtureCore)
```

Comment:
It would be good to define the `val modules` in BuildSrc so that `Module` can also be imported in a usage similar to `libs`.

Reply:
> Module can also be imported in a usage similar to libs.
 
Could you explain that in a little more detail?
What is difference between `object modules` and `val modules`?

Reply:
As shown below, it meant that it would be good to use the `Modules` as `modules`. I thought there would be uniformity when we used both the `libs` and `modules` in lowercase letters

```kotlin
val modules = Modules

object Modules : Iterable<Module> {
    private val modules = mutableListOf<Module>()

    val api = module(path = "order-api")
    val autoconfigure = module(path = "order-autoconfigure")
    val batch = module(path = "order-batch")
    val crypto = module(path = "order-crypto")
    val log = module(path = "order-log")
    val externalClient = module(path = "order-external-client")
    val jpa = module(path = "order-jpa")
    val json = module(path = "order-json")
    val kafka = module(path = "order-kafka")
    val lock = module(path = "order-lock")
    val redis = module(path = "order-redis")
    val support = module(path = "order-support")
    val testFixture = module(path = "order-test-fixtures")
    val xlt = module(path = "order-xlt")

    private fun module(path: String, name: String = ":lcp-$path"): Module =
        Module(path, name).also { modules.add(it) }

    override fun iterator(): Iterator<Module> =
        modules.iterator()
}

testImplementation(modules.testFixtureCore)   // Before Modules.testFixtureCore
testImplementation(modules.testFixtureEntity) // Before Modules.testFixtureEntity
testImplementation(libs.java.persistence.api)
testImplementation(libs.hibernate)
testImplementation(libs.h2)
```

Reply:
I have found that `modules` is defined in gradle (`DependencyHandlerDelegate.getModules()`)

<img width="756" alt="CleanShot 2022-09-20 at 21 26 30@2x" src="https://user-images.githubusercontent.com/22140938/191257209-ab016f62-69ea-44b9-bad4-fe1e80934527.png">

I could not find a way to solve this problem. 😢

Reply:
Oh... it's already been declared in gradle. Then there's nothing we can do. Thank you for checking